test/src/test/java/hudson/cli/CLITest.java

```diff
@@ -97,7 +96,7 @@ public void interrupt() throws Exception {
         r.jenkins.setSecurityRealm(r.createDummySecurityRealm());
         r.jenkins.setAuthorizationStrategy(new MockAuthorizationStrategy().grant(Jenkins.ADMINISTER).everywhere().to("admin"));
         FreeStyleProject p = r.createFreeStyleProject("p");
-        p.getBuildersList().add(new SleepBuilder(TimeUnit.MINUTES.toMillis(5)));
+        p.getBuildersList().add(new SleepBuilder(Long.MAX_VALUE));
```

Comment:
Using `Long.MAX_VALUE` to be consistent with other tests and to more clearly indicate the intention that this is meant to run "forever" before being interrupted.